rfcs/0086-create-secret.md

```diff
@@ -0,0 +1,64 @@
+---
+feature: New Syntax Create Secret
+authors:
+  - "TabVersion"
+start_date: "2024/03/18"
+---
+
+# Introducing SECRETs for Secure Storage of Sensitive Information
+
+## Background
+
+This RFC proposes a new syntax and implementation for securely storing and referencing sensitive information,
+such as database passwords and API keys, in SQL statements using a SECRET catalog.
+
+The primary goals are to enhance the security of production environments by encrypting sensitive data, applying access controls, 
+and preventing abuse of API keys by developers.
+
+## Proposed Syntax
+
+```sql
+-- create secret
+create secret <secret-name> from <expr>
```

Comment:
```suggestion
create secret <secret-name> as <expr>
```

Why using `from`? `AS` might be more consistent with other create statements, such as CTAS.